invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Home/Home.js

```diff
@@ -10,19 +10,20 @@ import { MostRecentBooks } from './components/MostRecentBooks';
 import { MostRecentEbooks } from './components/MostRecentEbooks';
 import { default as config } from './config';
 import { goTo } from '../../../history';
+import { SearchBar } from '../../../common/components/SearchBar';
 
 export default class Home extends Component {
   constructor(props) {
     super(props);
     this.state = { query: '' };
-    this.updateQuery = this.updateQuery.bind(this);
+    this.updateSearchQuery = this.updateSearchQuery.bind(this);
```

Comment:
we can avoid the `bind` and just change the function to an arrow function:
```
updateSearchQuery = (...) => {}
```

Reply:
👍